db/seeds.rb

```diff
@@ -4,9 +4,15 @@
 service = Service.find_by url: 'localhost'
 unless service
   service = FactoryGirl.create :service, name: 'dev stuff', url: 'localhost'
-  item_type = FactoryGirl.create :item_type, name: 'TEST_ITEM_TYPE', creator: service
+
+  test_type = FactoryGirl.create :item_type, name: 'TEST_ITEM_TYPE', creator: service
+  10.times { FactoryGirl.create :item, reservable: true, item_type: test_type }
+  four_seater = FactoryGirl.create :item_type, name: '4 Seat', creator: service
```

Comment:
I also notice that we don't have any non-reservable items in our seed data - could potentially be helpful.